test/spec/Feature/Query/PgSafeUpdateSpec.hs

```diff
@@ -37,40 +137,248 @@ spec =
+        get "/safe_delete_items"
+          `shouldRespondWith`
+            [json|[
+              { "id": 1, "name": "item-1", "observation": null }
+            , { "id": 2, "name": "item-2", "observation": null }
+            , { "id": 3, "name": "item-3", "observation": null }
+            ]|]
+
+        request methodDelete "/safe_delete_items?id=gt.0"
+          [("Prefer", "tx=commit"), ("Prefer", "count=exact")] mempty
           `shouldRespondWith`
             ""
             { matchStatus  = 204
             , matchHeaders = [ matchHeaderAbsent hContentType
-                             , "Content-Range" <:> "*/3" ]
+                             , "Content-Range" <:> "*/3"
+                             , "Preference-Applied" <:> "tx=commit" ]
             }
 
+        get "/safe_delete_items?order=id"
+          `shouldRespondWith`
+            [json|[]|]
+
+        request methodPost "/rpc/reset_items_tables"
+          [("Prefer", "tx=commit")]
+          [json| {"tbl_name": "safe_delete_items"} |]
+          `shouldRespondWith` ""
+          { matchStatus  = 204 }
+
 disabledSpec :: SpecWith ((), Application)
 disabledSpec =
   describe "Disabling pg-safeupdate" $ do
-    context "Full table update" $
-      it "does not update and does not throw error if no condition is present" $
-        request methodPatch "/unsafe_update"
-            [("Prefer", "count=exact")]
-            [json| {"name": "New name"} |]
+    context "Full table update" $ do
+      it "works if no condition is present" $ do
+        get "/unsafe_update_items"
+          `shouldRespondWith`
+            [json|[
+              { "id": 1, "name": "item-1", "observation": null }
+            , { "id": 2, "name": "item-2", "observation": null }
+            , { "id": 3, "name": "item-3", "observation": null }
+            ]|]
+
+        request methodPatch "/unsafe_update_items"
+            [("Prefer", "tx=commit"), ("Prefer", "count=exact")]
+            [json| {"name": "updated-item"} |]
           `shouldRespondWith`
             ""
-            { matchStatus  = 404
+            { matchStatus  = 204
+            , matchHeaders = [ matchHeaderAbsent hContentType
+                             , "Content-Range" <:> "0-2/3"
+                             , "Preference-Applied" <:> "tx=commit" ]
+            }
+
+        get "/unsafe_update_items?order=id"
+          `shouldRespondWith`
+            [json|[
+              { "id": 1, "name": "updated-item", "observation": null }
+            , { "id": 2, "name": "updated-item", "observation": null }
+            , { "id": 3, "name": "updated-item", "observation": null }
+            ]|]
+
+        request methodPost "/rpc/reset_items_tables"
+          [("Prefer", "tx=commit")]
+          [json| {"tbl_name": "unsafe_update_items"} |]
+          `shouldRespondWith` ""
+          { matchStatus  = 204 }
+
+      it "works wihtout filters, taking only the first item in the json array body" $ do
```

Comment:
Hm, this test here seems unrelated to pg-safeupdate. Doesn't it belong to the UpdateSpec, on the bulk patch section?